### PR TITLE
build(MF-9): configured eslint

### PR DESCRIPTION
**Pull request by KaterynaZhuvak**

### CODE REVIEW

**Comment by KaterynaZhuvak** [comment]
I just removed the file. If it is incorrect, I will think then what to do next.

File: .prettierrc
Comment (Sk3ll) [inline .prettierrc:1]: This file shouldn't delete or change

**Review by Sk3ll — CHANGES_REQUESTED** [review]

**Comment by Sk3ll** [comment]
Please change PR's name to _build(MF-9): configured eslint_. It shoud look like in Yuriy

File: .eslintrc.cjs [hunk 1]
```diff
@@ -1,19 +1,25 @@
+const {resolve} = require('node:path');
+
+const project = resolve(__dirname, 'tsconfig.json');
+
 module.exports = {
   root: true,
-  env: { browser: true, es2020: true },
+  env: {browser: true, es2020: true},
```
Comment (Sk3ll) [inline .eslintrc.cjs]: Please delete this line, should be better if you installed extends `@vercel/style-guide/eslint/browser`. It has more rules.

```suggestion
  extends: [
    require.resolve('@vercel/style-guide/eslint/browser'),
    require.resolve('@vercel/style-guide/eslint/react'),
    require.resolve('@vercel/style-guide/eslint/typescript'),
    'plugin:tailwindcss/recommended',
  ],
```

File: tsconfig.json [hunk 2]
```diff
@@ -21,5 +21,5 @@
     "noFallthroughCasesInSwitch": true
   },
   "include": ["src"],
-  "references": [{ "path": "./tsconfig.node.json" }]
```
Comment (Sk3ll) [inline tsconfig.json]: this is bad rule, when we delete this spacing between brackets

**Review by Sk3ll — CHANGES_REQUESTED** [review]
Please change name of pull request from `Mf-9/ESLint` to `build(MF-9): configured eslint`

File: .eslintrc.cjs [hunk 3]
```diff
@@ -1,19 +1,28 @@
+const {resolve} = require('node:path');
```
Comment (Sk3ll) [inline .eslintrc.cjs]:
```suggestion
const { resolve } = require('node:path');
```

File: README.md [hunk 4]
```diff
@@ -1 +0,0 @@
-
```
Comment (Sk3ll) [inline README.md:1]: roll back this file

**Review by Sk3ll — CHANGES_REQUESTED** [review]

**Review by Sk3ll — APPROVED** [review]

**Pull request merged**